etcdserver/api/membership/errors.go

```diff
@@ -21,10 +21,12 @@ import (
 )
 
 var (
-	ErrIDRemoved     = errors.New("membership: ID removed")
-	ErrIDExists      = errors.New("membership: ID exists")
-	ErrIDNotFound    = errors.New("membership: ID not found")
-	ErrPeerURLexists = errors.New("membership: peerURL exists")
+	ErrIDRemoved         = errors.New("membership: ID removed")
+	ErrIDExists          = errors.New("membership: ID exists")
+	ErrIDNotFound        = errors.New("membership: ID not found")
+	ErrPeerURLexists     = errors.New("membership: peerURL exists")
+	ErrMemberPromoted    = errors.New("membership: aleady a voting member")
+	ErrMemberNotPromoted = errors.New("membership: can not promote now")
```

Comment:
ErrLearnerNotReady = errors.New("membership: learner member is not sufficiently caught up")

Reply:
changed err msg